Portfolio/index.html

```diff
@@ -1,16 +1,72 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />

```

Comment:
The 'img' class div is unnecessary. You can apply styling directly to the 'img' element using CSS selectors.